public/index.js

```diff
@@ -1087,6 +1103,14 @@
                                   await response.text() + "'";
 
         this.parentNode.insertAdjacentElement('beforebegin', successMsg);
+
+        // adding a delay so user can see the success enrollment before it
+        // updatesthe user transaction screen immediately.
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
add a delay from when the user sees a success message to updating user transaction live